.idea/runConfigurations.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same as above, not a manual change that I made, however, the line appears to have been moved from vcs.xml and placed here